components/Sections/SectionsBuilder.tsx

```diff
@@ -22,6 +24,28 @@ export async function PageSectionsBuilder({ pagesId }: PageSectionsBuilder) {
   );
 }
 
+interface GroupSectionsBuilderProps {
+  groupsId: number;
+}
+
+export async function GroupSectionsBuilder({
```

Comment:
The only difference the two specific sections builder have is the usage of different prisma tables and different search fields in the `where` object. Maybe we could (in the future) create a generic `SectionsBuilder` that takes any intermediate sections table and the name of the field as a prop.